docs/reference/ingest/apis/enrich/enrich-stats.asciidoc

```diff
@@ -121,6 +121,10 @@ The amount of time in milliseconds spent fetching data from the cache on success
 `misses_time_in_millis`::
 (Long)
 The amount of time in milliseconds spent fetching data from the enrich index and updating the cache, on cache misses only.
+
+`size_in_bytes`::
```

Comment:
Don't we also usually document the human readable version (`size` in this case)?

Reply:
Although having said that, I see that we don't do that for `hits_time_in_millis` or `misses_time_in_millis` for this API.

Reply:
Yeah I thought about that as well, but couldn't find an example/case where we already do that, so I decided to leave it like this.